components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -731,65 +977,203 @@ export class AppCommandParser {
             }
             switch (f.type) {
             case AppFieldTypes.DYNAMIC_SELECT:
+                if (f.multiselect && Array.isArray(values[f.name])) {
+                    const options: AppSelectOption[] = [];
+                    const commandValues = values[f.name] as string[];
+                    for (let i = 0; i < commandValues.length; i++) {
```

Comment:
We can have slightly cleaner syntax here

```suggestion
                    for (const value of commandValues) {
```